java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java

```diff
@@ -225,6 +236,49 @@ public void run() {
         int executionresult = -10;
         final InputOutput ioput = getInputOutput();
 
+        MavenProxySupport mps = clonedConfig.getProject().getLookup().lookup(MavenProxySupport.class);
+        if (mps != null) {
+            MavenProxySupport.ProxyResult res;
```

Comment:
how about moving it to line 244 where it can be instanlty initialized?

Reply:
fixed in 5d31b280c9d8